src/monitors.js

```diff
@@ -46,7 +58,12 @@ module.exports.spotRequestFulfilled = (monitor, series, provisionerId, region, a
     instanceId,
     time,
   });
-  monitor.measure(`${region}.${az}.${instanceType}.${workerType}.spot.filled`, time);
+  monitor.count('overall.spot.died.count', 1);
+  monitor.measure('overall.spot.died.time', time);
+  monitor.count(`${region}.${az}.${instanceType}.overall.spot.filled.count`, 1);
+  monitor.measure(`${region}.${az}.${instanceType}.overall.spot.filled.time`, time);
+  monitor.count(`${region}.${az}.${instanceType}.worker.${workerType}.spot.filled.count`, 1);
+  monitor.measure(`${region}.${az}.${instanceType}.worker.${workerType}.spot.filled.time`, time);
```

Comment:
Particularly I suspect many of the metrics are interesting for a instance type across regions...